sdk/scala-sdk/src/main/scala/com/akkaserverless/scalasdk/ComponentOptions.scala

```diff
@@ -16,5 +16,5 @@
 
 package com.akkaserverless.scalasdk
 
-//FIXME see if needed
+//FIXME see if needed, maybe should be moved to impl
```

Comment:
if it is moved to impl we should (also) include `withForwardHeaders` and `forwardHeaders` in the more concrete options interfaces, such as EventSourcedEntityOptions

Reply:
yes will do

Reply:
These interfaces still extend ComponentOptions in impl, so should be good?